ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -122,7 +122,7 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
   }, {
     key: 'complianceTraining',
     completionPropsKey: 'trainingCompleted',
-    title: 'Complete Ethics training',
+    title: <span><i>All of Us</i> Responsible Conduct of Research Training</span>,
```

Comment:
Nit: i just came across while cleaning up some code <AouTitle/> maybe we can use it here too

Reply:
Sorry, not sure I understand. What can we maybe use here?

Reply:
There's an <AoUTitle/> element (I assume) but that specifically puts the text "_All of Us_ Research Program", which won't work here.

Reply:
My bad by editing the comment i just ended up deleting the Tag itself. But scout is right since it has research we cannot use but **not part of this ticket** it would be great to have a <i>All Of Us</i> so that in case of change we do not have to go to All the pages and do it